lesson_05/homework_5.2.py

```diff
@@ -0,0 +1,39 @@
+# Given list of tuples (name, surname, age, profession, City location)
```

Comment:
Please one pull request per one hometask  - do not mix all in single Pull Request